### ISSUE TITLE
error on login and signup

### ISSUE THREAD

**Issue by inordeck** [cg923/trans-friendly#1]
since you met and exceeded all the project requirements

i'm seeing the following error on login and logout:
TypeError: Cannot read property 'firstChild' of null

but i also reviewed and searched for firstChild in your code and could not find a reference to it in any of the files. i'm not sure what is throwing the error, but it is the only one i saw in the whole site.

**Comment by cg923** [comment]
Yeah, this is a problem with Google Maps.  I have to load one of their javascript files and the best place to do that is either in my header or footer... but if you don't actually place a Google Map on your page, you get this error.

So on my login and sign up pages it complains about google maps because the script is called in the footer but never used anywhere on the page.

The only way I can think to fix this would be to reorganize my footer to exclude any of my "body" HTML so that it wouldn't needlessly include this script, but then what's the point of my footer?

So, I guess my answer is: I understand why this is happening and it doesn't seem like a big deal and fixing it would actually create more work.  I won't be closing this issue because this is definitely a bit of a copout of an answer, but until something else occurs to me I'm going to leave it be.

Thank you!
